types/addons-linter/index.d.ts

```diff
@@ -0,0 +1,148 @@
+// Type definitions for addons-linter 6.15
```

Comment:
This package will need to be updated to the new layout.

Reply:
Sorry for the inconvenience. I have updated to the new layout.